### ISSUE TITLE
Warning: componentWillUpdate is deprecated and will be removed in the next major version.

### ISSUE THREAD

**Issue by DavyGuedes** [awesomejerry/react-native-qrcode-svg#61]
Hi. I'm just reporting this warning. But the component remains functional in my environment. 

"Warning: componentWillUpdate is deprecated and will be removed in the next major version. Use componentDidUpdate instead."
"Please update the following components: QRCode"

`react-native -v`
```bash
react-native-cli: 2.0.1
react-native: 0.60.3
```
`Environment`
```
Ubuntu 19.04
Open JDK 1.8.0_212, build vendor: Azul Systems
Device (emulated): Pixel 3 XL with Android Pie x86 image
```

**Comment by honter36** [comment]
I can fix that, let me a few days

**Comment by SpicyPete** [comment]
Let us know when it's in, looking forward to the fix

**Comment by SpicyPete** [comment]
@honter36 Any update on this fix?

**Comment by honter36** [comment]
Hi, sorry for my delay... I've been busy. In this weekend I'll send the pull request

**Comment by awesomejerry** [comment]
@honter36 is everything ok? If you're too occupied, I'll make the change.

**Comment by Iskander508** [comment]
Should be fixed in 5.2.0
